### ISSUE TITLE
Imap move/copy does not work when mailbox prefixed with server name

### ISSUE THREAD

**Issue by markatom** [php-mail-client/Client#10]
When I try to move a mail to mailbox named Test, an exception is thrown:

```
Cannot copy mail to mailbox 'Test': [TRYCREATE] UID COPY Invalid mailbox name
```

My mailbox is hosted on wedos.cz. I was further investigating why is this happening and I figured out that if the mailbox name is not prefixed with a server name (e.g. {foo.bar.net:993/imap/ssl}), it works.

The problematic line of code is here https://github.com/greeny/MailLibrary/blob/master/MailLibrary/Drivers/ImapDriver.php#L396

I do not know if the server name is required or if it could be omitted. Maybe some configuration option/switch removing the server name should be added.

**Comment by greeny** [comment]
Yeah, i thought about this too. The problem is, that some hosts prefix mailboxes with server name, but some does not. And also on one host, some mailboxes are prefixed and some are not.

MailLibrary does not want to show these server names to public, so it automatically removes these parts. But since the part is optional, library cannot know, if some mailbox should be prefixed or not. I think it would be good to implement some hash with `real mailbox name => shortified mailbox name` map and use this for translating mailbox names. But problem still persist on mailbox creation - should it be prefixed or not?

Have you found some article about this? Because I did not.
